gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/endpoint/mfa/MFAChallengeEndpoint.java

```diff
@@ -411,6 +413,7 @@ private void verify(FactorProvider factorProvider, FactorContext factorContext,
                             }
                         },
                         error -> {
+                            logger.info("Challenge failed for user {}", factorContext.getUser().getId());
```

Comment:
```suggestion
                            logger.error("Challenge failed for user {}", factorContext.getUser().getId());
```

Reply:
its not an error IMHO but normal flow :)